azimuth/plots/dataset_warnings.py

```diff
@@ -432,113 +434,69 @@ def class_representation(
 
 
 def create_histogram_mean_std(
-    train_count: np.ndarray,
-    eval_count: np.ndarray,
-    threshold_mean: float,
-    threshold_std: float,
-    train_stats: Tuple[np.ndarray, np.ndarray],
-    eval_stats: Tuple[np.ndarray, np.ndarray],
-    divergence_mean: Optional[float],
-    divergence_std: Optional[float],
-    label_name: Optional[str] = None,
+    hist_per_split: Dict[DatasetSplitName, np.ndarray],
+    agg_per_split: Dict[DatasetSplitName, Dict[Agg, float]],
+    divergence_per_agg: Optional[Dict[Agg, float]] = None,
 ) -> PlotSpecification:
     """Create the histogram traces and annotations for each label (and "all").
 
     Args:
-        train_count: Histogram values for the selected label in the train set.
-        eval_count: Histogram values for the selected label in the evaluation set.
-        threshold_mean: Alert value.
-        threshold_std: Alert value.
-        train_stats: Mean and std dev of the train.
-        eval_stats: Mean and std dev of the evaluation set.
-        divergence_mean: Difference in the mean. None means one distribution is empty.
-        divergence_std: Difference in the std dev. None means one distribution is empty.
-        label_name: Name of the label, when sending 'None', the traces are generated for 'all'.
+        hist_per_split: Histogram values for the selected label per split.
+        agg_per_split: Mean and std dev for the selected label per split.
+        divergence_per_agg: Difference in the mean and std dev. None means one distribution is
+            empty or we are generating the plot for 'all' labels.
 
     Returns:
         Plot for one label.
 
     """
     fig = go.Figure()
 
-    train_count_norm = train_count / max(sum(train_count), 1)
-    eval_count_norm = eval_count / max(sum(eval_count), 1)
+    hist_normalized = dict()
+    for split in hist_per_split.keys():
+        hist_normalized[split] = hist_per_split[split] / max(sum(hist_per_split[split]), 1)
```

Comment:
Also using a dict comprehension
```suggestion
    hist_normalized = {split: hist / max(sum(hist), 1) for split, hist in hist_per_split.items()}
```

Reply:
Right.. I'm not used to dict comprehension, and I forgot I could use them that way..  🤦‍♀️